ngraph/python/TUTORIAL.md

```diff
@@ -0,0 +1,216 @@
+### Using nGraph's Python API
+
+nGraph is OpenVINO's graph manipulation library, used to represent neural network models in the form of a computational graph. Using nGraph's Python APIs, you can create, inspect and manipulate computational graphs. You can also use the API to set runtime properties of the graph, which will be used by OpenVINO during inference.
+
+#### nGraph namespace module
+
+nGraph has a large set of mathematical operations, chosen to cover most ops used in popular neural network models. The `ngraph` module is the basic namespace used to expose commonly used API methods. You can inspect the `ngraph` module or call `help` on any method for usage information.
+
+```python
+import ngraph as ng
+dir(ng)
+help(ng.space_to_batch)
+```
+
+
+#### Creating a simple graph
+
+nGraph's Python API can be used to create computational models. Let's say you want to create a model which represents the following formula:
+
+    y = ReLU(x + b)
+
+You can use the following code to create parameters `x` and `b` and create the computational graph representing the formula:
+
+```python
+import numpy as np
+import ngraph as ng
+
+parameter_x = ng.parameter(shape=[2, 2], dtype=np.float32, name="x")
```

Comment:
In C++ documentation we use tags like this https://github.com/openvinotoolkit/openvino/blob/master/docs/examples/example_ngraph_utils.cpp#L9 to copy code snippets from source code to documentation. In C++ it helps to avoid syntax errors because we build this code. But in case of python I'm not sure that this tags will help.